pandas/io/excel/_xlwt.py

```diff
@@ -59,25 +59,33 @@ def __init__(
 
         if encoding is None:
             encoding = "ascii"
-        self.book = xlwt.Workbook(encoding=encoding, **engine_kwargs)
-        self.fm_datetime = xlwt.easyxf(num_format_str=self.datetime_format)
-        self.fm_date = xlwt.easyxf(num_format_str=self.date_format)
+        self._book = xlwt.Workbook(encoding=encoding, **engine_kwargs)
+        self.fm_datetime = xlwt.easyxf(num_format_str=self._datetime_format)
```

Comment:
these are public?

Reply:
Thanks for catching, they are now deprecated as well.